snippets/binarySearch.go.tmpl

```diff
@@ -0,0 +1,37 @@
+{{/*
+    This CC is a snippet help search a given element x in sorted array. The array must be ascending.
+    The array input is $arr. Lenght of array is $length. Element need to search is $element
+
+    Made by Alikuxac#4177
+*/}}
+
+{{ $arr := cslice 2 5 6 7 9 10 15 }}
+{{ $element := 15 }}
+{{ $length := len $arr }}
+{{ $data := sdict "Found" true "List" $arr "Left" 0 "Right" (sub $length 1) "Value" $element }}
```

Comment:
probably want to default to those values for `Left` and `Right` in the template definition if they're not present so users don't have to provide it every time

Reply:
Okay, fixed